src/deprecation.h

```diff
@@ -10,7 +10,7 @@
 // Per https://zips.z.cash/zip-0200
 // Shut down nodes running this version of code, 16 weeks' worth of blocks after the estimated
 // release block height. A warning is shown during the 14 days' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 1392800;
+static const int APPROX_RELEASE_HEIGHT = 1396476;
```

Comment:
This release height occurs in the past. Is that OK? Is that to get an earlier EOS date?

Reply:
Correct; it was easier to do this than to change the weeks until EoS, which would have required a separate PR and another round of CI.

Reply:
I double-checked that it works out to Jan 10.

Reply:
I've also checked that this gives an EOS of 10 Jan.